### PR TITLE
Create two methods using Bubble Sort algorithm

### PR DESCRIPTION
**Pull request by anewman15**
In this PR, we create two methods using the Bubble Sort methodology.

- The first one, bubble_sort, sorts array of numbers in ascending order.
- The second method, bubble_sort_by, sorts an array of strings based on the increasing length of the items. The smallest string comes first and the largest comes last. In this method we made use of anonymous block and yield

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Team, you did an amazing job :+1: 
Just a few changes are required before this project can be approved:
* [x] Please kindly move the `bubble_sort_by` method into `bubble_sort.rb`  file so that both methods are in a single file.
* [x] No debugging instructions should be in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)

#### [OPTIONAL]
* It would be great to add a screenshot of your project to the README file. Also if you add a small instruction on how to clone and run your project, it would make README more descriptive.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      end
+      i += 1
+    end
+  end
+  sorted_arr
+end
+
+arr = [12, 10, 9, 7, 6, 5, 4, 2]
+p bubble_sort(arr)
+p arr
```
Comment (pelzolga123) [inline bubble_sort.rb]: These debugging instructions should be removed.

File: bubble_sort_by.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  sorted_arr
+end
+
+arr = %w[hello hiya hey hi]
+method_value = bubble_sort_by(arr) do |left, right|
+  left.length - right.length
+end
+
+p method_value
+p arr
```
Comment (pelzolga123) [inline bubble_sort_by.rb]: These debugging instructions should be removed.

**Comment by anewman15** [comment]
@pelzolga123, Thanks! We'[ve made the changes

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  TEAM 👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**